### PR TITLE
add Sys::Info::Driver::Unknown + Sys::Info::Driver::Linux extensions to Perl v5.34.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3164.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/219f2cb4d1a364d47184a4aeecd686c8 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/4236375bca086db0c140655928275b4d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3575.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/cfbe4a45d4c1aa30165535459804fafe for a full test report.

**Comment by branfosj** [comment]
See comment at https://github.com/easybuilders/easybuild-easyconfigs/pull/15191#issuecomment-1079685839

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3112.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1207bf585858a25cda63b9290bd9cee2 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7fb0cc44211f576012b25984c7dc1a26 for a full test report.

File: easybuild/easyconfigs/p/Perl/Perl-5.34.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -1804,6 +1804,16 @@ exts_list = [
...
+        'source_tmpl': 'Sys-Info-Driver-Unknown-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BU/BURAK/'],
+        'checksums': ['02408843c8e36ea3d507e9f33fee48d6908543829ebe320f13d1bfe76af31e09'],
+    }),
+    ('Sys::Info::Driver::Linux', '0.7905', {
+        'source_tmpl': 'Sys-Info-Driver-Linux-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BU/BURAK/'],
+        'checksums': ['899c329bd3508ec5849ad0e5dadfa7c3679bbacaea9dda12404a7893032e8b7b'],
+    }),
 ]
```
Comment (branfosj) [inline easybuild/easyconfigs/p/Perl/Perl-5.34.0-GCCcore-11.2.0.eb:1817]:
```suggestion
    ('Unix::Processors', '2.046', {
        'source_tmpl': 'Unix-Processors-%(version)s.tar.gz',
        'source_urls': ['https://cpan.metacpan.org/authors/id/W/WS/WSNYDER/'],
        'checksums': ['3973ebdc44682c9c15c776f66e8be242cb4ff1dd52caf43ff446b74d4dccca06'],
    }),
]
```

Sorry - one more needed for #15191

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--skip"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3112.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/960454c8237578678f313005dc2b3d41 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c7f896e2b32491f52ae743fca157260f for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**